regolith/helpers/a_expensehelper.py

```diff
@@ -87,11 +87,10 @@ def db_updater(self):
         # dates
         if rc.begin_date:
             begin_date = date_parser.parse(rc.begin_date).date()
-            date = date_parser.parse(rc.begin_date).date().strftime('%Y-%m-%d')
+            date_string = rc.begin_date
         else:
             begin_date = dt.date.today()
-            date = begin_date
-            date = dt.date.today().strftime('%Y-%m-%d')
+            date_string = dt.date.today().strftime('%Y-%m-%d')
```

Comment:
why are you turning a date into a string, then back into a date later?  Just keep everything as date type.  

Also, don't change begin_date to date.  These two things have different meanings so keep it as date.  I think this should work:
```
        if rc.begin_date:
            begin_date = date_parser.parse(rc.begin_date).date()
        else:
            begin_date = dt.date.today()

Reply:
done